packages/core-contracts/contracts/mocks/utils/OwnableParent.sol

```diff
@@ -0,0 +1,12 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../utils/Ownable.sol";
+
+contract OwnableParent is Ownable {
```

Comment:
Could you rename this to `OwnableMock` pls?
We're following this pattern with all mocks.

Reply:
Will do, but I have a couple notes I'd like to be considered:

1. This particular module is not a mock. It inherits `Ownable` so it can be tested.
2. Could we move all test related modules under the corresponding test/ folders vs being inside the production folders?

Reply:
1. I understand the subtlety but IMO we should not refine the naming conventions for now. Atm, anything that needs to be modified to be testable is being added the word "Mock".
2. No, because you wouldn't be able to compile them.